rcl/src/rcl/init.c

```diff
@@ -82,6 +82,9 @@ rcl_init(
   // Zero initialize rmw context first so its validity can by checked in cleanup.
   context->impl->rmw_context = rmw_get_zero_initialized_context();
 
+  // Store the allocator.
+  context->impl->allocator = allocator;
```

Comment:
The allocator is also stored below (L154). I think we should remove that redundant logic.

Reply:
Right, I already fixed it.